packages/project-access/src/project/cap.ts

```diff
@@ -156,7 +156,7 @@ export async function getCapModelAndServices(
         services = services.map((value) => {
             return {
                 name: value.name,
-                urlPath: uniformUrl(value.urlPath),
+                urlPath: uniformUrl(value.endpoints?.find((val) => val.kind === 'odata')?.path ?? value.urlPath),
```

Comment:
Would be nice to improve readability 👍 

```suggestion
const { endpoints, urlPath } = value;
const odataEndpoint = endpoints?.find((endpoint) => endpoint.kind === 'odata');
const endpointPath = odataEndpoint?.path ?? urlPath;
return { ... urlPath: uniformUrl(endpointPath) ...};

```

Reply:
Thank you, i have added the change as suggested.